source/class/qx/tool/cli/commands/Test.js

```diff
@@ -20,7 +20,20 @@ const path = require("path");
 const process = require("process");
 
 /**
- * Compiles the project and serves it up as a web page
+ * Compiles the project, serves it up as a web page (default, can be turned off),
```

Comment:
With this approach registered test do not need an explicit event. What's about listening for the runaTest event here and execute a runTest method for all registered tests?

Reply:
Can you explain (maybe with code)? I don't fully understand...

Reply:
Will add a change to pr later ok?

Reply:
Sure!

Reply:
Or accept-as-is and create new PR`? Might be easier to follow.